README.md

```diff
@@ -7,18 +7,15 @@ Calibration tools used for autonomous driving
 - Ubuntu22.04
 - Ros Humble
 
-
 ## Installation procedures
 
-After cloning pilot-auto(private repository) or [autoware](https://github.com/tier4/autoware), execute the following commands:
+After installing [autoware](https://github.com/tier4/autoware) (please see [source-installation](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/) page), execute the following commands:
 
 ```bash
-mkdir src
-vcs import src < autoware.repos
-cd src/autoware
-git clone git@github.com:tier4/CalibrationTools.git
-cd ../../
-vcs import src < src/autoware/calibration_tools/calibration_tools.repos
+cd autoware
+wget https://raw.githubusercontent.com/tier4/CalibrationTools/tier4/universe/calibration_tools.repos
```

Comment:
For the humble branch, instead of
`https://raw.githubusercontent.com/tier4/CalibrationTools/tier4/universe/calibration_tools.repos`

shouldn't it be:
`https://raw.githubusercontent.com/tier4/CalibrationTools/humble/calibration_tools.repos`?

Reply:
@knzo25 I will merge humble branch to tier4/universe after this PR is merged.
So this is OK.